test/unit_test/ccb_parser_test.cc

```diff
@@ -24,93 +24,106 @@ BOOST_AUTO_TEST_CASE(ccb_parse_label)
   p.parse_name = v_init<substring>();
 
   {
-	  CCB::label label;
-	  parse_label(lp, &p, "ccb shared", label);
-	  BOOST_CHECK_EQUAL(label.explicit_included_actions.size(), 0);
-	  BOOST_CHECK(label.outcome == nullptr);
-	  BOOST_CHECK_EQUAL(label.type, CCB::example_type::shared);
-	  lp.delete_label(&label);
+    CCB::label label;
+    // Memset is used because in practice these labels are always Calloced.
+    // This is dangerous though and we should REALLY let this object have a constructor.
+    memset(&label, 0, sizeof(label));
```

Comment:
Can you use the scoped calloc mechanism that we use in VW for this? In other words:

```
auto ptr = scoped_calloc...()
CCB::label& label = *ptr;
```